db/models/cohorts.js

```diff
@@ -0,0 +1,63 @@
+const { query } = require('../index');
+
+module.exports = {
+  addCohort: (cohort) => {
+    return query(`
+      INSERT INTO cohorts (cohort_name, phase)
+      VALUES ('${cohort.cohort_name}', '${cohort.phase}')
+    `).then(res => {
+      return query(`
+        SELECT * FROM cohorts WHERE cohort_name='${cohort.cohort_name}'
+      `).then(res => res.rows[0])
+        .catch(err => err);
+    }).catch(err => err)
+  },
+  updateCohort: async(cohortId, newCohortInfo) => {
+    //update cohort
+    console.log("In controller");
+    try {
+      let update = await query(`
+      UPDATE cohorts SET (cohort_name, phase) = (
+        '${newCohortInfo.cohort_name}',
+        '${newCohortInfo.phase}'
+      ) WHERE id = ${cohortId}
+    `);
+
+    if (update.rowCount) {
+      console.log(`Updated cohort ${cohortId}`)
+    } else {
+      return "unable to update cohort record"
+    }
+    } catch (error) {
+      console.log(error);
+      return error;
+    }
+
+    //retrieve updated cohort
+    try {
+      let cohort = await module.exports.getCohortById(cohortId);
+      return cohort;
+    } catch(error) {
+      console.log(error);
+      return error;
+    }
+  },
+  getAllCohorts: async() => {
```

Comment:
I've not used async/await with try/catch blocks.  Do you have any reason to believe that it will misbehave without an await statement in the catch block in the event of an error?  I'll research.  I suspect not but it seems like a point of learning that I'm interested in.

Reply:
I checked it out.  Learn a new thing every day.